docs/getting_started.md

```diff
@@ -1,187 +1,284 @@
 # Getting started
-## Download
+This getting started realizes a scenario such as following image.
+
+1. User updates  the temperature attribute of Room entity using curl.
+2. Orion notifies to the thermometer.py as a function in Meteoroid. (HTTP POST)
+3. The function writes messages according to the value of temperature to result.
+4. User can confirm results using Meteoroid CLI
+
+![getting-started-scenario](./images/getting-started-scenario.png)
 
-```bash
-git clone https://github.com/OkinawaOpenLaboratory/fiware-meteoroid.git --recursive && cd fiware-meteoroid/
-```
 
 ## Install
 
-OpenWhisk must be running to build Meteoroid.
+### Download Meteoroid
+Download the fiware-meteoroid repository using git.
 
-### Install OpenWhisk
+```bash
+git clone https://github.com/OkinawaOpenLaboratory/fiware-meteoroid.git --recursive && cd fiware-meteoroid/
+```
+
+### Install [OpenWhisk](https://github.com/apache/openwhisk)
+[OpenWhisk](https://github.com/apache/openwhisk) must be running to build Meteoroid.
 
 ```bash
 cd fiware-meteoroid/docker/openwhisk-devtools/docker-compose
 make quick-start
-
 ```
 
 ### Install [Meteoroid CLI](https://github.com/OkinawaOpenLaboratory/fiware-meteoroid-cli)
```

Comment:
I think the CLI installation should be done after `docker-compose up` of the Meteoroid core.

Reply:
Done